docs/README.adoc

```diff
@@ -0,0 +1,99 @@
+= ShiftStack CI
+
+Each OpenShift on OpenStack continuous integration test job runs on one of two clouds:
+
+* MassOpenCloud: OSP 13
+* Vexxhost (private cloud offering): OpenStack based on 16
+
+Currently, we have three different test suites:
+
+* *Parallel*: The Conformance test suite, executed parallelizing tests
+* *Serial*: The Conformance test suite, executed one job at a time
+* *Early*: Smoke tests
+
+Currently, our jobs all target the same infrastructure:
+
+* Control Plane (and bootstrap) nodes boot from a 25GB high-performance Cinder volume
+* Control Plane: 3 VMs with:
+** 8 vCPUs
+** 16 GB RAM
+** 25 GB disk
+* Compute Nodes: 3 VMs with:
+** 8 vCPUs
+** 16 GB RAM
+** 25 GB disk
+* OpenShift SDN
+* Installer-provisioned network
+* FIP-powered connectivity (we use AWS Route53 as the external DNS)
+* The OpenStack cloud is using a valid HTTPS certificate (no cacert in clouds.yaml)
+* Swift as a backend for cluster-image-registry-operator
+
+.ShiftStack periodics
+|===
+|Installer |Test suite |Job name |OpenStack version 
+
+|IPI
+|Parallel
+|https://testgrid.k8s.io/redhat-openshift-ocp-release-4.6-informing#release-openshift-ocp-installer-e2e-openstack-4.6[release-openshift-ocp-installer-e2e-openstack-4.6]
+|OpenStack 13
```

Comment:
It would probably be better to document on which cloud the tests are running than trying to document the OpenStack version it's running on.